src/Web/WorkerController.php

```diff
@@ -0,0 +1,146 @@
+<?php
+/**
+ * WorkerController class file.
+ *
+ * @author Petra Barus <petra.barus@gmail.com>
+ * @author Adinata <mail.dieend@gmail.com>
+ */
+
+namespace UrbanIndo\Yii2\Queue\Web;
+
+use Yii;
+
+/**
+ * WorkerController is a web controller class that fetches work from queue and
+ * then run the job.
+ * The motivation comes from the HHVM limitation for running PHP terminal script.
+ * 
+ * @author Petra Barus <petra.barus@gmail.com>
+ * @author Adinata <mail.dieend@gmail.com>
+ */
+class WorkerController extends \yii\web\Controller
+{
+    
+    /**
+     * @var boolean
+     */
+    public $enableCsrfValidation = false;
+    
+    /**
+     * The default queue to use when there is no queue param sent in the request.
+     * @var string|array
+     */
+    public $defaultQueue = 'queue';
+    
+    /**
+     * The name of the request param to get the name of the queue component to use.
+     * This will check the POST first and the GET.
```

Comment:
The documentation is ambigous. What I received is if there is no post parameter named queue it will check get parameter, while what actually happened if it is a post parameter get from post else from get